editor.planx.uk/src/routes/team.tsx

```diff
@@ -65,7 +76,16 @@ const routes = compose(
           });
         }
 
-        useStore.getState().setFlowSlug(slug);
+        const stateFlowID = useStore.getState().id;
+
+        const flowData = await fetchDraftFlattenedFlowData(flow.id);
+
+        useStore.getState().setFlow({
+          id: stateFlowID,
+          flow: flowData,
+          flowName: flow.name,
+          flowSlug: slug,
+        });
```

Comment:
Question - I'm not sure I understand why this file or query has to change at all? Can we still find/load a flow based on it's slug?

If the issue is that `flowName` is not being set properly, and that's why this query needs to additionally return `name` now, then I think this is a good example where we want to call `useStore.getState().setFlowSlug()` & `useStore.getState().setFlowName()` individually, rather than this full `setFlow` which requires `id` & `data`. 

`fetchDraftFlattenedFlowData` is an expensive call and I think we want to avoid it on a `Team` route like this!

Reply:
Yeah we can definitely run everything individually, the only reason I added the ``fetchDraftFlattenedFlowData`` was because for ``SetFlow`` to work, it needed the flow data in a certain form which wasn't available.

I'll switch it back to individually setting the state.

Reply:
**For the original comment regarding the Section component**

I have added it and the ``flowName`` is shown correctly. Looks like it just pulls it in from the Global Store as ``flowName`` so I couldn't track anywhere for it going wrong.

I'll make the changes mentioned above for ``fetchDraftFlattenedFlowData`` and see if this fixes the e2e testing